.vib/jenkins/cypress/cypress/integration/jenkins_spec.js

```diff
@@ -0,0 +1,94 @@
+/// <reference types="cypress" />
+import { random, checkErrors } from './utils';
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.contains('.alert-danger').should('not.exist');
+  cy.get('div#tasks');
+
+  cy.get('[href$="/logout"]').click();
+  cy.get('form[name*="login"]');
+});
+
+it('should be possible to create a new Git Project', () => {
+  cy.login();
+  cy.visit('/view/All/newJob');
+
+  cy.fixture('items').then((item) => {
+    cy.get('#name').type(`${item.freestyleProject.name}-${random}`);
+    cy.contains(item.freestyleProject.type).click();
+    cy.contains('button', 'OK').should('be.enabled').click();
+    cy.contains('Error').should('not.be.visible');
+
+    cy.get('.radio-block-start').contains('Git').click();
+    cy.get("input[name*='url']").type(item.freestyleProject.repositoryURL);
+    cy.contains('button', 'Save').click();
+    cy.contains('h1', item.freestyleProject.name);
+  });
+
+  // As a new Git project is created in every execution, the next build
+  // will always be the first one.
+  const nextBuildNumber = 1;
+  cy.get("a[title='Build Now']").click();
+  cy.contains(`#${nextBuildNumber}`, { timeout: 30000 });
+
+  cy.get("a[title='Build Now']");
+  // Depending on the setup, the node where to execute the build needs to be
+  // provisioned, which can take up some time
+  cy.get(`a[href$='/${nextBuildNumber}/'][class*='display-name']`, {
+    timeout: 120000,
+  }).click();
+
+  cy.fixture('items').then((item) => {
+    cy.visit(
+      `/job/${item.freestyleProject.name}-${random}/${nextBuildNumber}/`
+    );
+    cy.contains(`Build #${nextBuildNumber}`);
+    cy.contains('Build has been executing for');
+  });
+});
+
+it('should be possible to register a new user', () => {
+  cy.login();
+  cy.visit('/securityRealm/addUser');
+
+  cy.contains('Create User');
+  cy.fixture('users').then((user) => {
+    cy.get('#username').type(`${user.newUser1.username}-${random}`);
+    cy.get("input[name='password1']").type(user.newUser1.password);
```

Comment:
```suggestion
    cy.get("[name='password1']").type(user.newUser1.password);
```
Same bellow this code, in all 3 inputs.